VotingApplication/VotingApplication.Web/Scripts/Directives/ResultsTable/ResultsTable.html

```diff
@@ -0,0 +1,15 @@
+﻿<md-list>
+    <div ng-repeat="result in resultBreakdown">
+        <md-list-item ng-click="showVoters=!showVoters">
```

Comment:
I _think_ the official guidelines are that `md-list-item` is a container, and thus should not have an `ng-click`, it should be handled by a button. That being said, I also think the demo docs show this usage as an example, and that there are also issues with dividers etc. showing if there's a button filling the whole list-item.